src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -271,8 +291,22 @@ AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(c
     VerifyOrReturnError(attestationNonceSpan.data_equal(attestationNonce),
                         AttestationVerificationResult::kAttestationNonceMismatch);
 
+    AttestationVerificationResult error;
     ByteSpan certificationDeclarationPayload;
-    return ValidateCertificationDeclarationSignature(certificationDeclarationSpan, certificationDeclarationPayload);
+    error = ValidateCertificationDeclarationSignature(certificationDeclarationSpan, certificationDeclarationPayload);
+    VerifyOrReturnError(error == AttestationVerificationResult::kSuccess, error);
+    // TODO: Retrieve these IDs from Basic Information Cluster
+    uint16_t clusterVendorId  = 0xFFF1;
+    uint16_t clusterProductId = 0x8000;
+
+    uint16_t dacProductId = 0xFFFFu; // not specified
+    uint16_t paiProductId = 0xFFFFu; // not specified
```

Comment:
This is where having had an input argument would have been great: instead of more arguments, a single metadata struct would be passed.

Reply:
Addressed in d0f7bb1